.github/workflows/ci.yml

```diff
@@ -220,6 +220,11 @@ jobs:
     - name: Install rust stable
       uses: dtolnay/rust-toolchain@stable
 
+    - name: Cleanup Docker
+      continue-on-error: true
+      run: |
+        docker kill $(docker ps -q)
```

Comment:
isn't there a chance this kills containers that are still in use?

Reply:
No, there's a mutex lock on every run for every box, so no chance of overlap.